src/views/conference/2018/index/index.jsx

```diff
@@ -78,6 +78,7 @@ var ConferenceSplash = React.createClass({
                                 </tr>
                             </tbody>
                         </table>
+                        <FormattedMessage id='conference-2018.registrationDate' />
```

Comment:
in order to be properly formatted, this should be wrapped in a `<p></p>` tag

Reply:
Additionally, there is a mismatch between the id here, and the id in the JSON file – here, it's `registrationDate`, and in the JSON file it's `registrationTime`.

Reply:
Can You explain more please?

Reply:
for the second thing i have to fix?

Reply:
@redmangospros I'm not sure which of the comments you're referring to, so I'll try to elaborate on both:
1. On other parts of the page, `FormattedMessage` components are wrapped in `<p>` tags, such as [here](https://github.com/LLK/scratch-www/pull/1748/files#diff-dd06e5d0ea78659a510800bf77c83c00L84). This line needs to be wrapped in that tag as well, in order to be formatted correctly. Please see the link I provided in the previous sentence for an example of how to wrap it. The text should look different when you view it locally after applying that change.
2. the `id` field on the `FormattedMessage` here says `registrationDate`. However, if you look  in the JSON file below, you'll see the id written there is actually [`registrationTime`](https://github.com/LLK/scratch-www/pull/1748/files#diff-647c54b0a5fdb8b9443d35965a7370e9R14). This mismatch means the text is not able to show up on the page right now.

Let me know if that helps!